script.js

```diff
@@ -0,0 +1,164 @@
+//1
+const createTeam = (teamName, driverName, birthYear, car, sponsor, isAvailableDriver) => {
+    const team = {
+        teamName,
+        driverName,
+        birthYear,
+        car,
+        sponsor,
+        isAvailableDriver
+    };
+    return team
+}
+const myFirstTeam = createTeam('Rolling Stones', 'Ken Block', '1984', 'Ford', 'Red Bull', true)
+//console.log(myFirstTeam)
+
+//2
+const createNewUser = () => {
+    const userNew = {
+      login: "",
+      password: "",
+      city: "",
+      country: "",
+      gender: "",
+      age: "",
+    };
+  
+    for (const [key] of Object.entries(userNew)) {
+      let result = prompt([key]);
+      if (result === null || result === "") {
+        userNew[key] = EMPTY_VALUE;
```

Comment:
What is `EMPTY_VALUE`?
Should it be a string that indicates the fact that object prop is empty, if yes it should be string with `""/''`  if its some variable, it should be initialized